whoami/src/os/web.rs

```diff
@@ -1,18 +1,69 @@
 #[cfg(not(any(target_pointer_width = "32", target_pointer_width = "64")))]
 compile_error!("Unexpected pointer width for target platform");
 
+use wasm_bindgen::prelude::*;
+use web_sys::window;
+
+
+
+
+
+
+#[wasm_bindgen]
+pub fn browser() -> String {
```

Comment:
There's already code for getting the browser name in whoami:

https://github.com/ardaku/whoami/blob/v2/WASM.md#web-browser